### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mywaystar**
The current English in the README is approximate. I just reworked it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,44 +2,42 @@
...
+It uses [RocksDB](https://github.com/facebook/rocksdb) as a built-in database, to store documents and internal data. The key-value store power allow us to handle updates and queries with small memory and CPU overheads.
 
-You can [read the deep dive](deep-dive.md) if you want more informations on the engine, it describes the whole process of generating updates and handling queries.
+You can [read the deep dive](deep-dive.md) if you want more information on the engine, it describes the whole process of generating updates and handling queries.
 
-We will be proud if you send pull requests to help us grow this project, you can start with [issues tagged "good-first-issue"](https://github.com/Kerollmops/MeiliDB/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) to start !
+We will be proud if you submit pull requests. It will help to help to grow this project, you can start contributing by checking [issues tagged "good-first-issue"](https://github.com/Kerollmops/MeiliDB/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22). It a good start!
```
Comment (tpayet) [inline README.md]:
```suggestion
We will be proud if you submit issues and pull requests. You can help to grow this project and start contributing by checking [issues tagged "good-first-issue"](https://github.com/Kerollmops/MeiliDB/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22). It is a good start!
```

File: README.md [hunk 2]
```diff
@@ -2,44 +2,42 @@
...
 
-At the moment this is a library only, this means that binaries are not part of this repository but since I'm still nice I have made some examples for you in the `examples/` folder that works with the data located in the `misc/` folder.
+At the moment this project is only a library. It means that it's not prividing yet any binaries. To get started, we provided  some examples in the `examples/` folder that are made to work with the data located in the `misc/` folder.
```
Comment (tpayet) [inline README.md]:
```suggestion
The project is only a library yet. It means that there is no binary provided yet. To get started, you can check the examples wich are made to work with the data located in the `misc/` folder.
```

File: README.md [hunk 3]
```diff
@@ -2,44 +2,42 @@
...
 
-In a near future MeiliDB we be a binary like any database: updated and queried using some kind of protocol. It is the final goal, [see the milestones](https://github.com/Kerollmops/MeiliDB/milestones). MeiliDB will just be a bunch of network and protocols functions wrapping the library which itself will be published to https://crates.io, following the same update cycle.
+In a near future MeiliDB, we will provide a binary to execute this project as database, so you will be able to update and query it using a protocol. This will be our final goal, [see the milestones](https://github.com/Kerollmops/MeiliDB/milestones). At the end, MeiliDB will be a bunch of network protocols, and wrappers. We will publish the entire project on https://crates.io, following our usual update cycle.
```
Comment (tpayet) [inline README.md]:
```suggestion
MeiliDB will be a binary in a near future so you will be able to use it as a database out-of-the-box. We should be able to query it using a [to-be-defined](https://github.com/Kerollmops/MeiliDB/issues/38) protocol. This is our current goal, [see the milestones](https://github.com/Kerollmops/MeiliDB/milestones). In the end, the binary will be a bunch of network protocols and wrappers around the library - which will also be published on [crates.io](https://crates.io). Both the binary and the library will follow the same update cycle.
```

File: README.md [hunk 4]
```diff
@@ -2,44 +2,42 @@
...
-We made some tests on remote machines and found that we can handle with a dataset of near 280k products, on a server that cost 5$/month with 1vCPU and 1GB of ram and on the same index and with a simple query:
-
-- near 190 users with an average response time of 90ms
-- 150 users with an average response time of 70ms
-- 100 users with an average response time of 45ms
-
-Network is mesured, servers are located in amsterdam and tests are made between two different datacenters.
+We made some tests on remote machines and found that MeiliDB easily handles a dataset of near 280k products, on a $5/month server with a single vCPU and 1GB of RAM, running the same index, with a simple query:
 
+- **near 190 concurrent users with an average response time of 90ms**
```
Comment (tpayet) [inline README.md]:
```suggestion
- near 190 concurrent users with an average response time of 90ms
```

Comment (tpayet) [inline README.md]: this is not actually the benchmark we want to highlight

File: README.md [hunk 5]
```diff
@@ -2,44 +2,42 @@
...
 
 ```bash
 cargo run --release --example create-database -- test.mdb misc/kaggle.csv
 ```
 
-Once the command finished indexing the database should have been saved under the `test.mdb` folder.
+Once the command is executed, the index should be in the `test.mdb` folder.
 
-Now you can easily run the `query-database` example to check what is stored in it.
+You are now able to run the `query-database` example, to play with MeiliDB.
```
Comment (tpayet) [inline README.md]:
```suggestion
You are now able to run the `query-database` example and play with MeiliDB.
```

File: README.md [hunk 6]
```diff
@@ -2,44 +2,42 @@
 
 A _full-text search database_ using a key-value store internally.
 
-It uses [RocksDB](https://github.com/facebook/rocksdb) like a classic database, to store documents and internal data. The key-value store power allow us to handle updates and queries with small memory and CPU overheads.
+It uses [RocksDB](https://github.com/facebook/rocksdb) as a built-in database, to store documents and internal data. The key-value store power allow us to handle updates and queries with small memory and CPU overheads.
```
Comment (tpayet) [inline README.md]:
```suggestion
It uses [RocksDB](https://github.com/facebook/rocksdb) as the internal key-value store. The key-value store allows us to handle updates and queries with small memory and CPU overheads.
```

File: README.md [hunk 7]
```diff
@@ -2,44 +2,42 @@
...
 
-In a near future MeiliDB we be a binary like any database: updated and queried using some kind of protocol. It is the final goal, [see the milestones](https://github.com/Kerollmops/MeiliDB/milestones). MeiliDB will just be a bunch of network and protocols functions wrapping the library which itself will be published to https://crates.io, following the same update cycle.
+In a near future MeiliDB, we will provide a binary to execute this project as database, so you will be able to update and query it using a protocol. This will be our final goal, [see the milestones](https://github.com/Kerollmops/MeiliDB/milestones). At the end, MeiliDB will be a bunch of network protocols, and wrappers. We will publish the entire project on https://crates.io, following our usual update cycle.
 
 
 
 ## Performances
 
-_these informations have been made with a version dated of october 2018, we must update them_
+_these information are outdated  (October 2018) It will be updated soon_
```
Comment (tpayet) [inline README.md]:
```suggestion
_These information are outdated (October 2018). These will be updated soon_
```

**Comment by Kerollmops** [comment]
@mywaystar 
You will need to rebase on master to be able to merge it.
Can you take a look to reviews please ? 😃

**Comment by mywaystar** [comment]
It's a readme guys!

**Pull request closed without merging**